docs/pages/docs/walkthroughs/embedded-mode-with-sqlite-nextjs.mdx

```diff
@@ -132,7 +132,7 @@ Edit the `next.config.js` file in your project root with the following:
 - module.exports = {
 -   reactStrictMode: true,
 - }
- 
+
```

Comment:
These changes are the result of a `yarn format` on the full code base. Might as well land in this PR 🤷‍♂️